libs/client-api/src/collab_sync/plugin.rs

```diff
@@ -192,7 +192,7 @@ where
         trace!("queue awareness: {:?}", update);
       }
 
-      ClientCollabMessage::new_update_sync(update_sync)
+      ClientCollabMessage::new_awareness_sync(update_sync)
```

Comment:
@Horusiath could you please confirm whether it was a typo or intentional to set the `awareness update` as `update_sync` instead of `awareness_sync`?